components/header/index.styl

```diff
@@ -36,19 +36,24 @@ header
 
 .header-content
   height 100%
-  display flex
-  flex-direction column
-  justify-content space-around
+  display table
   z-index 501
   position absolute
   top 0
 
+.header-content-inner
+  display table-cell
+  height 100%
+  width 100%
+  vertical-align middle
+
 .header-text
-  width 85%
-  margin 10% auto
-  padding-top 30px !important
+  width 80%
+  margin 16% auto 0 auto
+  // padding-top 30px !important
```

Comment:
:gun: kill these comments :) 

Also, curious about the margin up there, if you are vertically centering with the table layout you shouldn't need this.